e2e/commands/snap.e2e.2.ts

```diff
@@ -701,4 +701,33 @@ describe('bit snap command', function () {
       expect(() => helper.command.catObject(fileObj)).to.not.throw();
     });
   });
+  describe('merge tags', () => {
```

Comment:
these changes are from another PR that's already merged to master... Due to overzealous re-basing on my part

Reply:
try merging from master, it might resolve these

Reply:
I did merge from master, said all was up to date, which is the strangest part of it. I think it's git confusion because of a rebase, which made it think that these files are included in the changes between the previous base and the new one. 
Or something like that.
But all of these changes already exist in master, so not sure why it's being displayed here as a change...